addons/stock/models/stock_picking.py

```diff
@@ -356,6 +356,7 @@ class Picking(models.Model):
         check_company=True,
         help="When validating the transfer, the products will be assigned to this owner.")
     printed = fields.Boolean('Printed')
+    signature = fields.Binary('Signature', help='Signature', copy=False, attachment=True)
```

Comment:
i saw in some recent commit that we should use field.Image an d not binary here because it adds some extra protection